### ISSUE TITLE
All branches currently broken

### ISSUE THREAD

**Issue by rynowak** [aspnet/JitBench#63]
I'm seeing excessive CPU usage and build failures on the CI due to the `dotnet store` step running for an indefinite amount of time.

Currently investigating, but all runs appear to be blocked.

**Comment by rynowak** [comment]
https://github.com/dotnet/cli/issues/6936

**Comment by rynowak** [comment]
It appears that I'm also running into this issue: https://github.com/dotnet/sdk/issues/1302
```
EXEC : error : compilation failed for "D:\k\JitBench\.temp\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation_2.0.0-preview1
-final\runtimopt\Microsoft.Extensions.Configuration.EnvironmentVariables.dll" (0x80004005) [D:\k\JitBench\CreateStore.pr
oj]
  Microsoft (R) CoreCLR Native Image Generator - Version 4.5.22220.0
  Copyright (c) Microsoft Corporation.  All rights reserved.

D:\k\JitBench\.dotnet\sdk\2.0.0-preview3-006478\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.CrossGen.targets(172,5): erro
r MSB3073: The command "D:\k\JitBench\.temp\Optimize\netcoreapp\crossgen.exe -readytorun -platform_assemblies_paths D:\k
\JitBench\.temp\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation_2.0.0-preview1-final\runtimeref;D:\k\JitBench\.temp\Optim
ize\netcoreapp -CreatePDB D:\k\JitBench\.temp\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation_2.0.0-preview1-final\runtim
esymbols\microsoft.extensions.configuration.environmentvariables\2.0.0-preview1-final\lib\netstandard1.3 D:\k\JitBench\.
temp\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation_2.0.0-preview1-final\runtimopt\Microsoft.Extensions.Configuration.En
vironmentVariables.dll" exited with code -2147467259. [D:\k\JitBench\CreateStore.proj]
```

**Comment by rynowak** [comment]
Resolved by 4c3a0179c776801565044c38118972df37031031

**Comment by rynowak** [comment]
The issue here is external to us. Using the metapackage really simplifies the graph of things that need to be added to the store, which solves the timeouts.
